### DISCUSSION TITLE
ArtifactUpload>Sunbird-RC>CertificateApi fails

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-RC/community#297] [Issues] [answered]
Hi team,

CSP-GCP

**github_release_tag** `release-5.1.0`

I have builded and artifact upload it fails. Below is the error.

`+ rsync -Lkr /var/lib/jenkins/workspace/ArtifactUpload/dev/Sunbird-RC/CertificateApi/private/ansible/inventory/dev/Sunbird-RC/* /var/lib/jenkins/workspace/ArtifactUpload/dev/Sunbird-RC/CertificateApi/ansible/inventory/env/
rsync: change_dir "/var/lib/jenkins/workspace/ArtifactUpload/dev/Sunbird-RC/CertificateApi/private/ansible/inventory/dev/Sunbird-RC" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1196) [sender=3.1.2]`

**Comment by beepdot** [comment, marked as answer]
@tsprasath Please create a folder named `Sunbird-RC`. You can copy the `Kubernetes` directory and name it as `Sunbird-RC` in your private repo.
